src/tools/make_func.y

```diff
@@ -145,7 +145,7 @@ func: type ID optional_ID '(' arg_list optional_default ')' ';' {
   char f_name[500];
   int i, len;
   if (min_arg == -1) min_arg = $5;
-  if (min_arg > 4) yyerror("min_arg > 4\n");
+  if (min_arg > 127) yyerror("min_arg > 127\n");
```

Comment:
It would be nicer if an english readable error were here too - such as:
"You are using more than 127 arguments in an efun error.".